shenyu-client/shenyu-client-motan/src/main/java/org/apache/shenyu/client/motan/MotanServiceEventListener.java

```diff
@@ -110,7 +110,7 @@ private void handler(final Object bean) {
             clazz = AopUtils.getTargetClass(bean);
         }
         String superPath = buildApiSuperPath(clazz);
-        MotanService service = clazz.getAnnotation(MotanService.class);
+        ShenyuMotanService service = clazz.getAnnotation(ShenyuMotanService.class);
```

Comment:
learn and use `org.springframework.core.annotation.AnnotatedElementUtils#findMergedAnnotation` to replace `java.lang.Class#getAnnotation`.